package.json

```diff
@@ -1,5 +1,6 @@
 {
   "devDependencies": {
+    "@oclif/tslint": "^3.1.1",
```

Comment:
This is a quality-of-life change for me, as this is what is used in the tslint.json file. I'm curious if it is an IDE issue on my end. I get no lint help without this.